#!/usr/bin/env python
PACKAGE = "rasp_imu_hat_6dof"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("yaw_calibration", double_t, 0, "Yaw Calibration", 0, -180, 180)

exit(gen.generate(PACKAGE, "rasp_imu_hat_6dof", "imu"))
